tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/main/function/ArgumentParserPositiveTest.java

```diff
@@ -158,10 +158,7 @@ public Object[][] decimalValues() {
         return new Object[][]{
                 {"10", "10"},
 //                {"-10.123", "-10.123"},
```

Comment:
We currently don't support negative decimal values as args as they can clash with the short command options. Eg. - `-Ckey=value` 
Shall we move this to negative tests and add a comment as well?
They might provide an `undefined CLI argument` error.

Reply:
Created an issue https://github.com/ballerina-platform/ballerina-lang/issues/38311 earlier for this since there are no opened issues to track the `TODO` item added to this test. So we can add the negative tests with that issue.